client/galaxy/scripts/apps/login.js

```diff
@@ -0,0 +1,39 @@
+
+var jQuery = require( 'jquery' ),
+    $ = jQuery,
+    PANEL = require( 'layout/panel' ),
+    _l = require( 'utils/localization' ),
+    PAGE = require( 'layout/page' );
+
+window.app = function app( options, bootstrapped ){
```

Comment:
They are. Most of the code in app-view, app-login, and app-analysis has been moved over and reworked. The files in mvc/app will be unnecessary.

I'm less concerned about whether the load order is friendly for tours and more concerned with the larger architecture questions of dependencies. We can work out issues with tours later. Loading all the apps at once is not a good idea and tying them to the layout should be avoided as well.